core/benchmarks/object-detection/benchmark_script.py

```diff
@@ -421,12 +475,13 @@ def run_benchmarking_analysis(
             (AnnotationType.POLYGON, AnnotationType.POLYGON),
         ],
         limits_to_test=[5000, 5000],
+        compute_detailed=False,
     )
-
     # run raster benchmark
     run_benchmarking_analysis(
         combinations=[
             (AnnotationType.RASTER, AnnotationType.RASTER),
         ],
-        limits_to_test=[500, 500],
+        limits_to_test=[500],
```

Comment:
Can we switch back to two runs?

Reply:
thank you! yes, should have changed that back